scripts/workflows/release/run-release-with-gates.cjs

```
@@ -0,0 +1,125 @@
+      env: process.env,
+    });
+    log('✅ Phase 4 release agent completed successfully');
+    return true;
+  } catch (err) {
+    log(`❌ Phase 4 release agent failed: ${err.message}`, 'ERROR');
+    throw err;
+  }
+}
+
+async function runWithGates() {
+  const dryRun = process.env.INPUT_DRY_RUN === 'true';
```

ESLint

[error] 'process' is not defined.

(no-undef)